ledger/test-common/BUILD.bazel

```diff
@@ -103,6 +103,7 @@ da_scala_dar_resources_library(
             "**/*Exception*.daml",
             "**/*Interface*daml",
             "**/Limits.daml",
+            "**/Foo.daml",
```

Comment:
Is `Foo.daml` added into here purely as a reference (i.e. it isn't actually needed) ?

Reply:
Choice observers are not supported in LF `1.8` so had to exclude it.
Benchtool uses 1.14 anyway